test/testfixture/query_functions.go

```diff
@@ -38,6 +39,16 @@ func (fxt *TestFixture) IterationByName(name string, spaceID ...uuid.UUID) *iter
 	return nil
 }
 
+// SpaceByName returns the first space that has the given name (if any).
+func (fxt *TestFixture) SpaceByName(name string) *space.Space {
```

Comment:
You can leave it in but with my suggestions you won't need it.

Reply:
yup !
please see https://github.com/fabric8-services/fabric8-wit/pull/1770/commits/75622623f469327dd41526bd841df5dae623e6a2